EOLib.Graphics/EOLib.Graphics.csproj

```diff
@@ -15,7 +15,7 @@
   <ItemGroup>
     <PackageReference Include="AutomaticTypeMapper" Version="1.4.1" />
     <PackageReference Include="PELoaderLib" Version="1.5.0" />
-    <PackageReference Include="MonoGame.Framework.DesktopGL" Version="3.8.0.1641" />
+    <PackageReference Include="MonoGame.Framework.DesktopGL" Version="3.8.1.303" />
```

Comment:
Side-effect of upgrading MonoGame is that I have a pre-compiled hair clip effect that needs to be recompiled for 3.8.1...and I don't remember how I compiled it in the first place lol

```
~/git/EndlessClient/bin/Debug/client/net6.0 [ethan@BEASTMODE] $  ./EndlessClient
WARNING: Unable to initialize the midi sound system. Background music will not play.
Unhandled exception. System.Exception: This MGFX effect is for an older release of MonoGame and needs to be rebuilt.
   at Microsoft.Xna.Framework.Graphics.Effect.ReadHeader(Byte[] effectCode, Int32 index)
   at Microsoft.Xna.Framework.Graphics.Effect..ctor(GraphicsDevice graphicsDevice, Byte[] effectCode, Int32 index, Int32 count)
   at Microsoft.Xna.Framework.Graphics.Effect..ctor(GraphicsDevice graphicsDevice, Byte[] effectCode)
   at EndlessClient.GameExecution.EndlessGame.LoadContent() in /home/ethan/git/EndlessClient/EndlessClient/GameExecution/EndlessGame.cs:line 154
   at Microsoft.Xna.Framework.Game.Initialize()
   at EndlessClient.GameExecution.EndlessGame.Initialize() in /home/ethan/git/EndlessClient/EndlessClient/GameExecution/EndlessGame.cs:line 112
   at Microsoft.Xna.Framework.Game.DoInitialize()
   at Microsoft.Xna.Framework.Game.Run(GameRunBehavior runBehavior)
   at Microsoft.Xna.Framework.Game.Run()
   at EndlessClient.GameExecution.EndlessGame.EndlessClient.GameExecution.IEndlessGame.Run()
   at EndlessClient.GameExecution.GameRunnerBase.RunGame() in /home/ethan/git/EndlessClient/EndlessClient/GameExecution/GameRunnerBase.cs:line 141
   at EndlessClient.Program.Main(String[] args) in /home/ethan/git/EndlessClient/EndlessClient/Program.cs:line 34
Aborted
```

Reply:
Oof that sucks. Hopefully you remember? 🤣